config/init_db.sql

```diff
@@ -1,30 +1,22 @@
 # This file is executed immediately after initializing a fresh data directory.
 
-###############################################################################
-# WARNING: This sql is *NOT* safe for production use,
-#          as it contains default well-known users and passwords.
-#          Care should be taken to change these users and passwords
-#          for production.
-###############################################################################
-
```

Comment:
Do we want to keep this warning? We no longer have static secrets in this file anymore, which is a really good thing so the current warning is no longer accurate.

Reply:
I think we can/should remove it.